integration/common/tests/sql/test_parser.py

```diff
@@ -178,6 +178,7 @@ def test_parse_simple_insert_into_select():
     assert sql_meta.out_tables == [DbTableMeta('table1')]
 
 
+@pytest.mark.skipif(provider() == "rust", reason="sqlparser-rs requires additional keyword")
```

Comment:
Lack of `INTO` keyword - sqlparser requires it, like Postgres.
Created PR for sqlparser-rs to support this: https://github.com/sqlparser-rs/sqlparser-rs/pull/473

I think it will get merged rather easily, since it's optional for multiple databases.

Also, added support for `UPDATE` statement in parser, so removed skip for update statement.